README.md

```diff
@@ -10,12 +10,10 @@ cmake -S . -B build -A Win32
 cmake --build build
 ```
 
-## xi_checker
+## Release notes
 
-Verifies DirectPlay and POL are installed and connect to a server using xiloader (installed in POL folder).
+#Windows:
+Requires VC2022 redist, included in https://aka.ms/vs/17/release/vc_redist.x86.exe
```

Comment:
Newline before content starts after a title in markdown

Reply:
Done